src/main/java/de/dhbw/karlsruhe/services/StartUpDialogService.java

```diff
@@ -1,29 +1,26 @@
 package de.dhbw.karlsruhe.services;
 
 import java.security.NoSuchAlgorithmException;
-import java.util.Scanner;
-
-import de.dhbw.karlsruhe.model.User;
+import de.dhbw.karlsruhe.models.User;
 
 public class StartUpDialogService {
 
-	UserService userService;
-	Scanner scanner;
+	private UserService userService;
 
 	public StartUpDialogService() {
 		userService = new UserService();
-		scanner = new Scanner(System.in);
 	}
 
-	public void signIn() {
+	public boolean signIn() {
 		boolean successfulSignedIn = false;
 
 		while (!successfulSignedIn) {
 			System.out.println("Do you already have an account? y/n");
-
-			String input = scanner.nextLine();
+			String input = ScannerService.getScanner().nextLine();
 			successfulSignedIn = signInProcess(input);
 		}
+
+		return successfulSignedIn;
```

Comment:
In dieser Methode werden die Änderungen aus der Pull-Request "11 - Loginverfahren angepasst" wieder rückgängig gemacht. So wie ich das sehen kann es wie in der anderen Pull-Request bleiben. Der Rückgabewert von signIn() wird nie gebraucht und ist sowieso immer true. Damit muss in der Main() Methode auch nicht die VAriable successfullLogin gesetzt werden.

Reply:
Das stimmt, habe die Änderungen wieder rückgängig gemacht. Der Status Parameter um zu zeigen, dass man eingeloggt ist wird nun direkt beim Aufruf der Login Methode gesetzt.